content/applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst

```diff
@@ -0,0 +1,189 @@
+   :align: center
+   :alt: The Record Production button on a work order.
+
+The tablet view defaults to showing the first work order for the *002* manufacturing order. Since
+this manufacturing order will not be completed immediately, back out of tablet view by clicking the
+:guilabel:`⬅️ (back)` button twice. Doing so will take you to the *001* order.
+
+To finish the *001* order, select the :guilabel:`Work Orders` tab and click the :guilabel:`tablet
+view` button located on the line of the next work order. Finally, complete the remaining work
+orders, then click :guilabel:`Mark As Done And Close MO` to close the manufacturing order.
+
+Once the remaining units are ready to be manufactured, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, then select the *002* order. Select the :guilabel:`Work
+Orders` tab and click the :guilabel:`tablet view` button located on the line of the first work
+order.
+
+If all of the units in the backorder will be completed immediately, simply click :guilabel:`Mark As
+Done And Close MO` after completing all of the work orders.
+
+If only some of the remaining units will be manufactured immediately, enter the number in the
+:guilabel:`Units` field at the top left of the tablet view, then click :guilabel:`Record Production`
+to create an additional backorder for the remaining units, with *003* at the end of its reference
+tag.
+
+The *002* backorder and *003* backorder can be completed by following the steps detailed in this
+section.
+
+.. tip::
+   It is also possible to create a backorder in the middle of a manufacturing order, when some but
+   not all of the work orders have already been completed. Doing so marks the completed work
+   order(s) as :guilabel:`Finished` on the backorder.
+
+   .. example::
+      A manufacturing order for four chairs requires the completion of two work orders: *Paint* and
+      *Assemble*. While the paint step can be completed immediately for all four chairs, there are
+      only enough screws to assemble two of them.
+
+      As a result, the employee responsible for producing the chairs begins by painting all four,
+      and marking the *Paint* work order as :guilabel:`Finished` for all of them. Then, they move on
+      to the *Assemble* work order. They assemble two of the four chairs, enter that number in the
+      :guilabel:`Units` field of the tablet view, and click :guilabel:`Record Production`.
+
+      A backorder manufacturing order is created for the remaining two chairs. On the backorder, the
+      *Paint* work order is already marked as :guilabel:`Finished`, and only the *Assemble* work
+      order is left to be completed.
+
+      Once more screws are available, the manufacturing employee assembles the remaining chairs and
+      clicks :guilabel:`Mark As Done And Close MO` to complete the *Assemble* work order and close
+      the backorder manufacturing order.
```

Comment:
I see what you did here and it makes sense. 
I think though we should undo the **example** embedding inside the **tip** admonition, since the formatting really doesn't look good. Having separate, distinct blocks still communicates the point clearly I think.

```suggestion
.. example::
   A manufacturing order for four chairs requires the completion of two work orders: *Paint* and
   *Assemble*. While the paint step can be completed immediately for all four chairs, there are only
   enough screws to assemble two of them.

   As a result, the employee responsible for producing the chairs begins by painting all four, and
   marking the *Paint* work order as :guilabel:`Finished` for all of them. Then, they move on to the
   *Assemble* work order. They assemble two of the four chairs, enter that number in the
   :guilabel:`Units` field of the tablet view, and click :guilabel:`Record Production`.

   A backorder manufacturing order is created for the remaining two chairs. On the backorder, the
   *Paint* work order is already marked as :guilabel:`Finished`, and only the *Assemble* work order
   is left to be completed.

   Once more screws are available, the manufacturing employee assembles the remaining chairs and
   clicks :guilabel:`Mark As Done And Close MO` to complete the *Assemble* work order and close the
   backorder manufacturing order.
```